packages/nbdime/tslint.json

```diff
@@ -1,58 +1,34 @@
 {
   "rules": {
-    "align": [
-      true,
-      "parameters",
-      "statements"
-    ],
+    "align": [true, "parameters", "statements"],
```

Comment:
Does the new prettier config still align with tslint config?

Reply:
As discussed today, you will check this. I haven't done it before to push new commits (interactive rebase).

Reply:
I have checked. There are some small discrepancies, but nothing major, so fine to leave it as it is.